pyproject.toml

```diff
@@ -66,6 +68,9 @@ dev = [
 ]
 test_extras = [
     "mcp; python_version >= '3.10'",
+    "datasets>=2.14.6",
```

Comment:
open for discussion - do you think it's better to have these go to `dev` or `test_extras`? My first impression is they belong to `dev` better, but not very sure

Reply:
We need this in `test_extras` but we can also add to `dev`